src/WaferProber-Accretech_UFseries/main.py

```diff
@@ -172,15 +183,63 @@ def get_current_subsite(self):
         current_subsite = ""
 
         return current_subsite
+    
+    def load_wafer(self, wafer_str):
+
+        self.print_info()
+
+        self.print_status()
+
+        self.prober.reset_alarm()
+
+        self.check_and_sense_wafers()
+
+        wafer_status = self.prober.request_wafer_status()
+        # a list of tuples containing cassette and wafer id
+        wafer_list = self.prober.get_waferlist_from_status(wafer_status)
 
+        wafer = wafer_str.replace("C", "").split("W")  # creates a list, e.g. [1,3]
+        wafer = tuple(map(int, wafer))
+
+        # wafer is a tuple with cassette number and wafer number.
+        # adding (3,) extends this tuple with the status where 3 means wafer on chuck 
+        if wafer + (3,) not in wafer_list:
+            # independent whether there is a wafer on the chuck we can use
+            # load_specified_wafer after check for sensed wafers
+            # If there is a wafer on the chuck, it will automatically be unloaded
+            self.prober.load_specified_wafer(*wafer)
+        else:
+            message_box(f"Wafer {wafer_str} is already on the chuck!", blocking=False)
+        
+    def unload_wafer(self):
+
+        self.print_info()
+
+        self.print_status()
+
+        self.prober.reset_alarm()
+
+        if self.prober.is_wafer_on_chuck():
+            self.prober.preload_specified_wafer(9, 99)
+        else:
+            message_box("There is no wafer on the chuck, that can be unloaded!", blocking=False)
+        
     def connect(self):
 
         # creating an AccretechProber instance that handles all communication
         self.prober = accretech_uf.AccretechProber(self.port)
         self.prober.set_verbose(self.verbosemode)
 
+        self.unique_identifier = "Driver_Accretech_UFseries_" + self.port_str
+        if not self.unique_identifier in self.device_communication:
+            self.device_communication[self.unique_identifier] = None
+
     def disconnect(self):
-        del self.prober  # this makes sure the event mechanism is disabled
+
+        if self.unique_identifier in self.device_communication:
+            self.prober.unregister_srq_event()  # this makes sure the event mechanism is disabled
+            del self.device_communication[self.unique_identifier]
+        del self.prober  
```

Comment:
Same comments as for the temperature driver

Reply:
same answer as for the temperature driver